hex/Hex2.cs

```diff
@@ -163,12 +173,16 @@ class Hex2 : UserControl {
             var charWidth = (int)(Font.Size * 2);
             var asciiStart = charWidth * numColumns;
             rect = new Rectangle(col * charWidth, row * 24, charWidth, 24);
-            Invalidate(rect);
+            Invalidate(rect);       
 
             charWidth = (int)Font.Size;
             asciiStart += charWidth;
             rect = new Rectangle(asciiStart + col * charWidth, row * 24, charWidth, 24);
             Invalidate(rect);
+
+            //redraws the whole section, not sure if there's a better way.
+            Invalidate();
```

Comment:
This is not a good solution. We need to recalculate the rect with respect to the window scrolling position.